benches/src/erc721.rs

```diff
@@ -57,9 +57,9 @@ pub async fn run(cache_opt: Opt) -> eyre::Result<Vec<FunctionReport>> {
     let token_3 = uint!(3_U256);
     let token_4 = uint!(4_U256);
 
-    let _ = receipt!(contract.mint(alice_addr, token_2))?;
-    let _ = receipt!(contract.mint(alice_addr, token_3))?;
-    let _ = receipt!(contract.mint(alice_addr, token_4))?;
+    _ = receipt!(contract.mint(alice_addr, token_2))?;
+    _ = receipt!(contract.mint(alice_addr, token_3))?;
+    _ = receipt!(contract.mint(alice_addr, token_4))?;
```

Comment:
```suggestion
    receipt!(contract.mint(alice_addr, token_2))?;
    receipt!(contract.mint(alice_addr, token_3))?;
    receipt!(contract.mint(alice_addr, token_4))?;
```
nit: not sure why ever made these redundant assignments haha